src/main/java/org/jfrog/hudson/pipeline/common/executors/XrayExecutor.java

```diff
@@ -39,7 +40,7 @@ public void execute() throws Exception {
         Log log = new JenkinsBuildInfoLog(listener);
         Credentials credentials = server.createCredentialsConfig().provideCredentials(build.getParent());
         ArtifactoryXrayClient client = new ArtifactoryXrayClient(server.getUrl(), credentials.getUsername(),
-                credentials.getPassword(), log);
+                credentials.getPassword(), credentials.getAccessToken(), log);
```

Comment:
Related to question I asked here - https://github.com/jfrog/build-info/pull/405#pullrequestreview-518456855
Why do we actually need to pass username, password and access token? Don't we use either username and password or access token?

Reply:
As I said there, with that, I don't have to check if username and password or access token were chosen.